src/lexer.jl

```diff
@@ -637,20 +637,20 @@ end
 
 # Parse a token starting with a quote.
 # A '"' has been consumed
-function lex_quote(l::Lexer)
```

Comment:
Yes I think that is worthwhile since we don't have to create the token object then, perhaps (`EMPTY_TOKEN`).